spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/ExpireSnapshotsSparkAction.java

```diff
@@ -184,11 +187,21 @@ public Dataset<FileInfo> expireFiles() {
 
       expireSnapshots.cleanExpiredFiles(false).commit();
 
-      // fetch metadata after expiration
-      Dataset<FileInfo> validFileDS = validFileDS(ops.refresh());
+      // determine expired files
```

Comment:
nit: We have the same comment ``determine expired files`` on different blocks now.

Reply:
Changed with your other code suggestion